model/metrics/dotnet/dotnet-health_checks.yaml

```
@@ -0,0 +1,49 @@
+groups:
+  - id: health_checks
+    prefix: health_checks
```

```suggestion
    prefix: aspnetcore.health_checks
```

Or
```suggestion
    prefix: dotnet.health_checks
```

otherwise this will be too generic health check metric semconv.